proto/axelar/evm/v1beta1/events.proto

```diff
@@ -50,3 +40,78 @@ message PollExpired {
     (gogoproto.nullable) = false
   ];
 }
+
+message ConfirmKeyTransferStarted {
+  string chain = 1
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+  bytes tx_id = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customtype) = "Hash",
+    (gogoproto.customname) = "TxID"
+  ];
+  bytes gateway_address = 3
+      [ (gogoproto.nullable) = false, (gogoproto.customtype) = "Address" ];
+  uint64 confirmation_height = 4;
+  vote.exported.v1beta1.PollParticipants participants = 5
+      [ (gogoproto.nullable) = false, (gogoproto.embed) = true ];
+}
+
+message ConfirmGatewayTxStarted {
+  bytes tx_id = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customtype) = "Hash",
+    (gogoproto.customname) = "TxID"
+  ];
+  string chain = 2
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+  bytes gateway_address = 3
+      [ (gogoproto.nullable) = false, (gogoproto.customtype) = "Address" ];
+  uint64 confirmation_height = 4;
+  vote.exported.v1beta1.PollParticipants participants = 5
+      [ (gogoproto.nullable) = false, (gogoproto.embed) = true ];
+}
+
+message ConfirmDepositStarted {
+  bytes tx_id = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customtype) = "Hash",
+    (gogoproto.customname) = "TxID"
+  ];
+  string chain = 2
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+  bytes deposit_address = 3
+      [ (gogoproto.nullable) = false, (gogoproto.customtype) = "Address" ];
+  bytes token_address = 4
+      [ (gogoproto.nullable) = false, (gogoproto.customtype) = "Address" ];
+  uint64 confirmation_height = 5;
+  vote.exported.v1beta1.PollParticipants participants = 6
+      [ (gogoproto.nullable) = false, (gogoproto.embed) = true ];
```

Comment:
Don't quite get what `embed` does?

Reply:
it allows us to call `event.PollID` instead of `event.Participants.PollID`, so it's just nicer to read